tests/utils.js

```diff
@@ -689,7 +689,6 @@ module.exports = {
         method: 'POST',
         body: { docs: docs }
       });
-
```

Comment:
I kind of like this better with the newline separating the other lines. Just my personal preference though.